src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -70,6 +70,7 @@ public void CanGetAllTimesAfterCreation()
             ValidateSetTimes(item, beforeTime, afterTime);
         }
 
+        [OuterLoop("Incurs long sleeps")]
```

Comment:
It should not sleep more than 1/1000 runs, so I'm curious to see a log where it does. Perhaps only on @sdmaclea distro?

Nevertheless no harm in outerlooping it. I should have just done it myself 😊

Reply:
> It should not sleep more than 1/1000 run

Won't it sleep every time if the file system doesn't support milliseconds?

Reply:
Right, but I have not seen that on our CI or official builds, which [cover 4 file systems](https://github.com/dotnet/corefx/issues/27662#issuecomment-375015547)

It does not work on OSX, but this test does not run on OSX

Reply:
By distro above, I should have said "filesystem for /tmp on whatever machine @sdmaclea is running on"